services/web/server/src/simcore_service_webserver/projects/_handlers_project_ports.py

```diff
@@ -85,14 +85,14 @@ async def _get_validated_workbench_model(
 routes = web.RouteTableDef()
 
 
-class _InputSchemaConfig:
+class _InputSchemaConfig(BaseModel):
```

Comment:
Be careful here . Note that the subclasses have **double** inheritance 
```python
class ... (_InputSchemaConfig, _ProjectIOBase):
   ...
```

The idea was that the first brings `Config` and the second `BaseModel`

I wonder if it would be better to do the following:


```python
from pydantic import BaseConfig, BaseModel

class _InputSchemaConfig(BaseConfig):
        allow_population_by_field_name = False
        extra = Extra.forbid
        allow_mutations = False

class ProjectInputUpdate(_ProjectIOBase):
    class Config(_InputSchemaConfig):
            ...
class ProjectInputGet(, _ProjectIOBase):
    label: str
    class Config(_OutputSchemaConfig):
        ...
```

which IMO reads very nice as :  ``ProjectInputUpdate`` is a `_ProjectIOBase` model, **configured** as an input schema ``_InputSchemaConfig ``

Reply:
you are right, I like it :+1: done